com.github.muriloventuroso.pdftricks.json

```diff
@@ -97,16 +103,26 @@
                 "--enable-zero-configuration"
             ],
             "sources": [{
-                    "type": "archive",
-                    "url": "https://imagemagick.org/archive/ImageMagick-7.1.0-61.tar.xz",
-                    "sha256": "4b4f9437666ef1a87df69b34bc8abdf7ad05d524c00dd1d63f6699e6ab1ed94e"
+                    "type": "git",
+                    "url": "https://github.com/ImageMagick/ImageMagick",
+                    "tag": "7.1.1-47"
```

Comment:
same here

Reply:
was easier to find the new tag there, but ok.